web_console/odc-about-web-terminal.adoc

```diff
@@ -15,6 +15,14 @@ Cluster administrators can access the web terminal in {product-title} 4.7 and la
 
 This terminal instance is preinstalled with common CLI tools for interacting with the cluster, such as `oc`, `kubectl`,`odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx`. It also has the context of the project you are working on and automatically logs you in using your credentials.
 
+The following new features have been introduced in the web terminal:
+
+* Adding multiple tabs - Support is provided for adding multiple tabs within the Web Terminal in the console. By default, a single tab appears when you open the Web Terminal. You can open multiple tabs by clicking the *+* button on the terminal.
+
+* Enable history - The new tab persists bash history, list of commands that were executed, within the terminal. You can use the terminal, leave it and return after some time but the terminal retains the bash history.
+
+* Increased timeout – The default timeout for the web terminal has been increased from 15 minutes to the duration of the browser window.
```

Comment:
Please replace the dash separators with periods or colons, as indicated in the IBM Style Guide section on "Punctuation in lists."

```suggestion
* Adding multiple tabs: Support is provided for adding multiple tabs within the Web Terminal in the console. By default, a single tab appears when you open the Web Terminal. You can open multiple tabs by clicking the *+* button on the terminal.

* Enable history: The new tab persists bash history, list of commands that were executed, within the terminal. You can use the terminal, leave it and return after some time but the terminal retains the bash history.

* Increased timeout: The default timeout for the web terminal has been increased from 15 minutes to the duration of the browser window.
```